tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -25,3 +25,5 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 ### [Changing personal drive quota on another user as admin is not possible](https://github.com/owncloud/ocis/issues/4325)
 - [apiSpaces/changeSpaces.feature:221](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/changeSpaces.feature#L221)
 
+### [create request for already existing user exits with status code 500 ](https://github.com/owncloud/ocis/issues/3516)
+- [apiGraph/createGroupCaseSensitive.feature:4](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/createGroupCaseSensitive.feature#L4)
```

Comment:
you have to mention every line of the examples table, not the line of the outline